content/en/docs/topics/monitoring-your-app.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Monitoring your app
+description: How to view telemetry data from your Spin apps running in SpinKube.
+weight: 13
+---
+
+This topic guide shows you how to configure SpinKube so your Spin apps export observability data. This data will export to an OpenTelemetry collector which will send it to Jaeger.
+
+## Prerequisites
+
+Please ensure you have the following tools installed before continuing:
+
+- A Kubernetes cluster running SpinKube. See the [installation guides](https://www.spinkube.dev/docs/install/) for more information
+- The [kubectl CLI](https://kubernetes.io/docs/tasks/tools/)
+- The [Helm CLI](https://helm.sh)
+
+## About OpenTelemetry Collector
+
+From the OpenTelemetry [documentation](https://opentelemetry.io/docs/collector/):
+>> The OpenTelemetry Collector offers a vendor-agnostic implementation of how to receive, process and export telemetry data. It removes the need to run, operate, and maintain multiple agents/collectors. This works with improved scalability and supports open source observability data formats (e.g. Jaeger, Prometheus, Fluent Bit, etc.) sending to one or more open source or commercial backends.
+
+In our case, the OpenTelemetry collector serves as a single endpoint to receive and route telemetry data, letting us to monitor metrics, traces, and logs via our preferred UIs.
+
+## About Jaeger
+
+From the Jaeger [documentation](https://www.jaegertracing.io/docs/):
+>> Jaeger is a distributed tracing platform released as open source by Uber Technologies. With Jaeger you can: Monitor and troubleshoot distributed workflows, Identify performance bottlenecks, Track down root causes, Analyze service dependencies
+
+Here, we have the OpenTelemetry collector send the trace data to Jaeger.
+
+## Deploy OpenTelemetry Collector
+
+First, add the OpenTelemetry collector Helm repository:
+
+```sh
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+helm repo update
+```
+
+Next, deploy the OpenTelemetry collector to your cluster:
+
+```sh
+helm upgrade --install otel-collector open-telemetry/opentelemetry-collector \
+    --set image.repository="otel/opentelemetry-collector-k8s" \
+    --set nameOverride=otel-collector \
+    --set mode=deployment \
+    --set config.exporters.otlp.endpoint=http://jaeger-collector.default.svc.cluster.local:4317 \
+    --set config.exporters.otlp.tls.insecure=true \
+    --set config.service.pipelines.traces.exporters[0]=otlp \
+    --set config.service.pipelines.traces.processors[0]=batch \
+    --set config.service.pipelines.traces.receivers[0]=otlp \
+    --set config.service.pipelines.traces.receivers[1]=jaeger
```

Comment:
These commands might need quotes in zsh.

Reply:
I see this error in zsh `zsh: no matches found: config.service.pipelines.traces.exporters[0]=otlp` and can't get it to work for some reason...

Reply:
Ok I just had to prefix the command with `noglob`

Reply:
I'm running zsh as well, and it looks like I'm able to run this when I paste the commands in a `.sh` with `#!/bin/bash`, but doesn't work when I have it run `#!/bin/zsh`. Does MacOS have access to bash?

Reply:
I'm not familiar enough with zsh vs bash to know why it's behaving this way...

Reply:
Is it unwieldy to escape the `[` and `]` with a backslash? It looks like this will work on both ZSH and BASH:

```sh
helm upgrade --install otel-collector open-telemetry/opentelemetry-collector \
    --set image.repository="otel/opentelemetry-collector-k8s" \
    --set nameOverride=otel-collector \
    --set mode=deployment \
    --set config.exporters.otlp.endpoint=http://jaeger-collector.default.svc.cluster.local:4317 \
    --set config.exporters.otlp.tls.insecure=true \
    --set config.service.pipelines.traces.exporters\[0\]=otlp \
    --set config.service.pipelines.traces.processors\[0\]=batch \
    --set config.service.pipelines.traces.receivers\[0\]=otlp \
    --set config.service.pipelines.traces.receivers\[1\]=jaeger

```

Reply:
I tested that on zsh and it worked 👍. It's a little ugly but I think it is the best solution to make sure it works in both shells.